tests/utils.ts

```diff
@@ -15,19 +15,43 @@ export async function createTmp() {
 /**
  * Abstraction for install, as the blueprint supports multiple package managers
  */
-export async function install({ cwd }: { cwd: string }) {
-  await execa('yarn', ['install', '--non-interactive'], { cwd });
+export async function install({ cwd, packageManager }: { cwd: string; packageManager: string }) {
+  if (packageManager === 'yarn') {
+    await execa('yarn', ['install', '--non-interactive'], { cwd });
+  } else {
+    try {
+      await execa(packageManager, ['install'], { cwd });
+    } catch (e) {
+      if (packageManager === 'pnpm') {
+        console.info('An error occurred. Are there still upstream issues to resolve?');
+        console.error(e);
+
+        return;
+      }
```

Comment:
It *seems* to me this special handling of pnpm here was probably mostly useful during development. Do we still need this, or can we just remove the catch-block?

Also, even if we want some special error message for pnpm, shouldn't this still throw? This is now returning "successfully" even when execa throws.

Reply:
We do need this, because the app-blueprint is not peer-ily correct. The start of resolving that is here:

https://github.com/babel/ember-cli-babel/pull/452

Reply:
Both pnpm and npm support disabling the strict peer dep checking via `.npmrc`. If that is the reason for your special error handling, we should just use that setting instead.

Reply:
given our discussion about doing some more involved maintenance on ember-cli-babel, and doing a major over there, I went ahead and did the _very specific error catching_ in this PR so we don't over-catch errors